src/coreclr/vm/arm64/asmhelpers.S

```diff
@@ -23,6 +23,14 @@ LEAF_ENTRY GetDataCacheZeroIDReg, _TEXT
     ret     lr
 LEAF_END GetDataCacheZeroIDReg, _TEXT
 
+#if defined(_MSC_VER)
```

Comment:
```suggestion
```
.S file is only used with gcc/clang

Reply:
ooh cool. Thanks for confirming it. I'll move the definition from .S file as it would fail on clang with `error: instruction requires: sve or sme`.

Reply:
You should be able to use `.arch_extension sve` in the .S file to get the SVE instruction to compile

Reply:
Done.